src/AppInstallerCLICore/ExecutionContext.h

```diff
@@ -52,9 +52,18 @@ namespace AppInstaller::CLI::Execution
         LogPath,
         InstallerArgs,
         CompletionData,
+        ContextFlag,
+        InstalledPackageVersion,
         Max
     };
 
+    // bit masks used as Context flags
+    namespace ContextFlag

```

Comment:
>namespace ContextFlag [](start = 4, length = 21)

enum class

Reply:
And then define some operators.

---
In reply to: [503506092](https://github.com/microsoft/winget-cli/pull/588#discussion_r503506092) [](ancestors = 503506092)